elements/rh-tile/test/rh-tile-group.spec.ts

```diff
@@ -0,0 +1,173 @@
+
+describe('<rh-tile>', function() {
+  let element: RhTileGroup;
+  const tileGroup = html`
+      <rh-tile-group radio>
+        <rh-tile checked>
+          <div slot="title">Title</div>
+          <h2 slot="headline"><a href="#top">Link</a></h2>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+          <div slot="footer">Suspendisse eu turpis elementum</div>
+        </rh-tile>
+    
+        <rh-tile>
+          <div slot="title">Title</div>
+          <h2 slot="headline"><a href="#top">Link</a></h2>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+          <div slot="footer">Suspendisse eu turpis elementum</div>
+        </rh-tile>
+    
+        <rh-tile>
+          <div slot="title">Title</div>
+          <h2 slot="headline"><a href="#top">Link</a></h2>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+          <div slot="footer">Suspendisse eu turpis elementum</div>
+        </rh-tile>
+      </rh-tile-group>`;
+
+  async function click(element: HTMLElement) {
+    const { x, y, width, height } = element.getBoundingClientRect();
+    const position = [x + width / 2, y + height / 2].map(Math.round) as [number, number];
+    await sendMouse({ type: 'click', position });
+  }
+
+  async function arrowDown() {
+    await sendKeys({ down: 'ArrowDown' });
+    await element.updateComplete;
+  }
+
+  async function enter() {
+    await sendKeys({ press: 'Enter' });
+    await element.updateComplete;
+  }
+
+  async function arrowUp() {
+    await sendKeys({ down: 'ArrowUp' });
+    await element.updateComplete;
+  }
+
+  function tiles(element) {
+    return [...element.querySelectorAll('rh-tile')];
+  }
+
+  function input(element, checked = false) {
+    return element?.shadowRoot?.querySelector(`input[type="radio"]${checked ? ':checked' : ''}`);
+  }
+
+  function inputs(element, checked = false) {
+    return tiles(element).map(tile => input(tile, checked)).filter(input=>input);
+  }
+
+  describe('simply instantiating', function() {
+    it('imperatively instantiates', function() {
+      expect(document.createElement('rh-tile-group')).to.be.an.instanceof(RhTileGroup);
+    });
+
+    it('should upgrade', async function() {
+      element = await createFixture<RhTileGroup>(html`<rh-tile-group></rh-tile-group>`);
+      const klass = customElements.get('rh-tile-group');
+      expect(element)
+        .to.be.an.instanceOf(klass)
+        .and
+        .to.be.an.instanceOf(RhTileGroup);
+    });
+  });
+
+  describe('radio tile', function() {
+    it('is accessible', async function() {
+      element = await createFixture<RhTileGroup>(tileGroup);
+      expect(element)
+        .to.be.an.accessible;
+    });
+
+    it('has radio buttons', async function() {
+      element = await createFixture<RhTileGroup>(tileGroup);
+      expect(tiles(element).length)
+        .to.equal(inputs(element).length);
+    });
+
+    describe('acts as a radio group', async function() {
```

Comment:
```suggestion
    describe('as a radio group', async function() {
```

combine this with the next sibling describe, since they use the same template. inline the template